app/admin/dashboard.rb

```diff
@@ -0,0 +1,16 @@
+ActiveAdmin.register_page "Dashboard" do
+  menu priority: 1, label: proc { I18n.t("active_admin.dashboard") }
```

Comment:
let's use single quotes for ruby code, unless you are doing string interpolation

Reply:
Great, i fixed it here: 82e36fb